wiki/Beatmapping/New_Combo/en.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - NC
+---
+
+# New combo
+
+*Main page: [Combo](/wiki/Beatmapping/Combo/)*
+
+*Not to be confused with [Nightcore](/wiki/Game_Modifiers/Nightcore/).*
+
+**New combos** (more commonly known as *NCs*) signal the start of a new [combo](/wiki/Beatmapping/Combo/). How new combos behave during gameplay depends on the currently selected [game mode](/wiki/Game_Modes/). They can be selected while [beatmapping](/wiki/Beatmapping/) in the [beatmap editor](/wiki/Beatmap_Editor) by clicking the `New Combo` button on the right side of the `Compose` tab, or by using the shortcut `Q`. New combos are an integral part to colourhaxing.
+
+## osu!standard
+
+In [osu!standard](/wiki/Game_Modes/osu!), each [hit object](/wiki/Hit_object/) has its own combo number and combo colour. When a new combo is started, the combo colour changes and the combo number resets to 1. This counts up for each following object.
+
+*Note: Spinners force the next hit object to start a new combo.*
```

Comment:
```suggestion
*Note: Spinners force the next hit object to start a new combo.*
```
Hatnotes must start off a paragraph or section

Reply:
A note is not a hatnote, it is a part of the [Notices](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria#notices). These are like hatnotes but they have their own set of rules to follow. Notices don't actually need to start off a section or be placed before any paragraphs in a section, just wherever they think it is good.

Reply:
Oh well that makes my life significantly easier